x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -153,6 +153,15 @@ export class TaskManagerPlugin
       this.logger.info(`TaskManager is identified by the Kibana UUID: ${this.taskManagerId}`);
     }
 
+    if (
+      this.config.claim_strategy === CLAIM_STRATEGY_MGET &&
+      this.config.poll_interval > MGET_DEFAULT_POLL_INTERVAL
+    ) {
+      this.logger.warn(
+        `Setting xpack.task_manager.poll_interval to a value greater than the default of ${MGET_DEFAULT_POLL_INTERVAL}ms may increase task latency and reduce overall throughput. Please remove this setting unless it is explicitly required.`
```

Comment:
```suggestion
        `Setting xpack.task_manager.poll_interval to a value greater than the default ( ${MGET_DEFAULT_POLL_INTERVAL} ms) can increase task latency and reduce overall throughput. Do not override the default unless it is explicitly required.`
```

Reply:
I'm not sure about the "explicitly required". How does one know it's required? Is it based on the speed of the system or the urgency of having a fast task turnaround?

Reply:
@lcawl from what I gather, the majority of users who configure this setting want a slower poll interval for some reason. I'm not sure why but the only advantage it brings to them is fewer queries to Elasticsearch when they're not running many background tasks. We've automated this behaviour recently so that we look for tasks every 500ms and back-off to every 3s when not necessary.

Reply:
It might be nice to callout the improvements since folks might not have known about them otherwise (and thus no longer need to override the default). For example: 

> By default, task manager polls occur every 500ms and scale to 3s as necessary. Setting xpack.task_manager.poll_interval to a value greater than the default (500 ms) can increase task latency and reduce overall throughput; it is not recommended.